packages/js/product-editor/src/components/editor/editor.tsx

```diff
@@ -29,7 +29,7 @@ import { ShortcutProvider } from '@wordpress/keyboard-shortcuts';
 // eslint-disable-next-line @typescript-eslint/ban-ts-comment
 // @ts-ignore No types for this exist yet.
 // eslint-disable-next-line @woocommerce/dependency-group
-import { FullscreenMode, InterfaceSkeleton } from '@wordpress/interface';
+import { InterfaceSkeleton } from '@wordpress/interface';
```

Comment:
Yep, maybe we can re consider this when we start working on the new design. Nor sure yet, but fullscreen can be a requested feature then.